README.md

```diff
@@ -0,0 +1,30 @@
+# scala-bootstrap
+
+This project is the support for the exercices. It can be run both in the command line and in an IDE.
+We recommend installing IntelliJ, but you can still use Scala-Eclipse IDE.
+
+## Install
+Make sure you have Java 1.8 or higher installed on your machine. If not, [follow the instructions](https://www.java.com/fr/download/)
+Install IntelliJ following the steps on the [website](https://www.jetbrains.com/idea/download/).
```

Comment:
These steps are summarized on https://www.scala-lang.org/download/, I guess you can add this link